portfolio/src/main/webapp/script.js

```diff
@@ -81,12 +81,48 @@ function createCommentElement(comment) {
   commentElement.classname = 'comment';
 
   const headerElement = document.createElement('span');
-  headerElement.innerHTML = "<h3>"+comment.posterName +"<h3>";
+  headerElement.innerHTML = "<h3>" + cleanseString(comment.posterName) +"<h3>";
 
   const bodyElement = document.createElement('span');
-  bodyElement.innerHTML = "<p>" + comment.comment +"<p>";
+  bodyElement.innerHTML = "<p>" + cleanseString(comment.comment) +"<p>";
 
   commentElement.appendChild(headerElement);
   commentElement.appendChild(bodyElement);
   return commentElement;
-}
\ No newline at end of file
+}
+
+function cleanseString(html) {
+  // Prevent html injection by replacing escape characters with plaintext
+  return html.replace(/</g, "&lt;").replace(/>/g, "&gt;");
+}
+
+google.charts.load('current', {'packages': ['corechart']});
+google.charts.setOnLoadCallback(drawChart);
```

Comment:
put these two lines at the top of the file

Reply:
Good idea. Why are these called outside of a function? (I emulated this from the walkthrough code)